urlshortener/extensions.py

```diff
@@ -1,5 +1,25 @@
 from flask_sqlalchemy import SQLAlchemy
 from flask_login import LoginManager
+from .db import setup_db
+import string
+from random import choice
+import azure.cosmos.exceptions as exceptions
 
 db = SQLAlchemy()
 login_manager = LoginManager()
+cosmos_db = setup_db()
```

Comment:
This is not necessarily wrong, but somewhat dangerous. When I see this line used locally (rather than called during initialization), I would assume that I can copy this line to get access to the db in my own update to this project. However, that will introduce a race condition since `setup_db` performs multiple sensitive db operations.